diff --git a/android/build.gradle b/android/build.gradle
index d6ccc18aab7927e69794f69af38844cfe04371e4..01f3a07f48d73dc3bebcc1efbabcb4ff78f532bd 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -15,5 +15,5 @@ android {
 }
 
 dependencies {
-  implementation 'com.github.Dimezis:BlurView:version-3.1.0'
+  implementation 'com.github.Dimezis:BlurView:3.2.0'
 }
diff --git a/build/BlurView.types.d.ts b/build/BlurView.types.d.ts
--- a/build/BlurView.types.d.ts
+++ b/build/BlurView.types.d.ts
@@ -1,5 +1,5 @@
 import type { RefObject } from 'react';
-import type { ViewProps, View } from 'react-native';
+import type { ViewInstance, ViewProps } from 'react-native';
 /**
  * Blur method to use on Android.
  *
@@ -24,4 +24,4 @@ export type BlurViewProps = {
      *
      * @platform android
      */
-    blurTarget?: RefObject<View | null>;
+    blurTarget?: RefObject<ViewInstance | null>;
@@ -69,4 +69,4 @@ export type BlurViewProps = {
 } & ViewProps;
 export type BlurTargetViewProps = {
-    ref?: RefObject<View | null>;
+    ref?: RefObject<ViewInstance | null>;
 } & ViewProps;
