diff --git a/dist/commonjs/native-internal/root.js b/dist/commonjs/native-internal/root.js
index 4e77f38f5c8f61621581161b67c682584f1ae029..06848764050f66d2645d9e503c9dde9f445637b1 100644
--- a/dist/commonjs/native-internal/root.js
+++ b/dist/commonjs/native-internal/root.js
@@ -24,7 +24,7 @@ const rootVariableFamily = () => {
     return obs;
   });
 };
-const rootVariables = exports.rootVariables = rootVariableFamily();
+const rootVariables = exports.rootVariables = globalThis.__react_native_css_root_variables ??= rootVariableFamily();
 const universalVariables = exports.universalVariables = rootVariableFamily();
 rootVariables("__rn-css-rem").set([[14]]);
 // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
diff --git a/dist/module/native-internal/root.js b/dist/module/native-internal/root.js
index 9800a999be235dfb20a78581b13842267bb8bd14..800504670fdca51835d26391bbd9e7cff0a642ea 100644
--- a/dist/module/native-internal/root.js
+++ b/dist/module/native-internal/root.js
@@ -20,7 +20,7 @@ const rootVariableFamily = () => {
     return obs;
   });
 };
-export const rootVariables = rootVariableFamily();
+export const rootVariables = globalThis.__react_native_css_root_variables ??= rootVariableFamily();
 export const universalVariables = rootVariableFamily();
 rootVariables("__rn-css-rem").set([[14]]);
 // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
diff --git a/src/native-internal/root.ts b/src/native-internal/root.ts
index e45a7d11b12e226999ab1f1defc37a7acb716792..1bbbd2e6279c2f63b2e2e1ed8e9a20dbb8f7e2c6 100644
--- a/src/native-internal/root.ts
+++ b/src/native-internal/root.ts
@@ -29,7 +29,8 @@ const rootVariableFamily = () => {
   });
 };
 
-export const rootVariables = rootVariableFamily();
+export const rootVariables =
+  globalThis.__react_native_css_root_variables ??= rootVariableFamily();
 export const universalVariables = rootVariableFamily();
 
 rootVariables("__rn-css-rem").set([[14]]);
